python/tests/test_event_reporter.py

```
@@ -329,7 +329,11 @@ async def test_post_event_failure_log_redacts_base64_audio(
             return_value=httpx.Response(500, text="boom")
         )
         with caplog.at_level(logging.ERROR):
-            await reporter.post_event(event)
+            # post_event now raises on failure so the bus can retry; this
+            # event's raw-dict messages make serialization itself fail, which
+            # is the in-flight exception path the redaction must also cover.
+            with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)